File: 2. Loop Problems/Loop Problems in c/p9.c
// Write a C program to check whether a number is palindrome or not

#include <stdio.h>

int main() {
    int n, temp, og, r_sum = 0;
    printf("Enter number: ");
    scanf("%d", &n);
    
    og = n; 

    while (n != 0)
    {
        temp = n % 10;  
        n = n / 10;     
        r_sum = r_sum * 10 + temp;  
    }

    
    if (og == r_sum)
    {
        printf("%d is a palindrome.\n", og);
    }
    else
    {
        printf("%d is not a palindrome.\n", og);
    }

    
}

File: 2. Loop Problems/Loop problems in python/p9.py
# // Write a python program to check whether a number is palindrome or not

n = int(input("Enter number: "))
reverse = int(str(n)[::  -1])
print(f" Reverse: {reverse}\n")

if n == reverse:
    print(f"{n} is a plaindrome")
else:
    print(f"{n} is a not plaindrome")
    